app/src/main/res/values/strings.xml

```diff
@@ -11,5 +11,13 @@
 
     <!-- Home Activity -->
     <string name="profile_floating_action_button">profile floating action button</string>
+    <!-- Intro Activity and viewPager Adapter-->
```

Comment:
not Intro Activity ( onBoarding Fragments) and no strings here used in ViewPagerAdapter Class